_docs-sources/pipelines/maintain/extending.md

```diff
@@ -50,27 +61,36 @@ infrastructure-deployer --aws-region "us-east-1" -- docker-image-builder build-d
+
+For example, a common configuration for `apply` jobs is to specify that this job can only run on the `main` branch:
+```tf
+allowed_apply_git_refs = ["main", "origin/main"]
+```
 
-### RefArch
+<Tabs groupId="deployment-type">
+<TabItem value="RefArch" label="RefArch" default>
 
-Open `shared/<your region>/mgmt/ecs-deploy-runner/terragrunt.hcl` and update the values the `allowed_apply_git_refs` attribute for the job configuration you would like to modify (either `terraform_planner_config` or `terraform_applier_config`).
+If you've deployed Pipelines as a part of your Reference Architecture, open `shared/<your region>/mgmt/ecs-deploy-runner/terragrunt.hcl` and update the values in the `allowed_apply_git_refs` attribute for the job configuration you would like to modify (either `terraform_planner_config` or `terraform_applier_config`).
 
 Run `terragrunt plan` to inspect the changes that will be made to your pipeline. Once the changes have been reviewed, run `terragrunt apply` to deploy the changes.
 
-### Standalone
+</TabItem>
+<TabItem value="Standalone" label="Standalone">
 
 If you've deployed Pipelines as a standalone framework using the `ecs-deploy-runner` service in the Service Catalog, you will need to locate the file in which you've defined a module block sourcing the `ecs-deploy-runner` service.
 
 By default, the `ecs-deploy-runner` service from the Service Catalog allows any git ref to be applied. After you locate the module block for `ecs-deploy-runner`, modify the `allowed_apply_git_refs` attribute for the job configuration that you would like to modify (either `terraform_planner_config` or `terraform_applier_config`).
 
 Run `terraform plan` to inspect the changes that will be made to your pipeline. Once the changes have been reviewed, run `terraform apply` to deploy the changes.
+</TabItem>
+</Tabs>
 
-## Adding script arguments
+## Adding scripts that can be run in Pipelines
 
 The `deploy-runner` Docker image for Pipelines only allows scripts within a single directory to be executed in the ECS task as an additional security measure.
 
-By default, the `deploy-runner` ships with three scripts - one to build HashiCorp Packer images, one to run `terraform plan` and `terraform apply`, and one to automatically update the value of a variable in a Terraform tfvars or Terragrunt HCL file.
+By default, the `deploy-runner` ships with three scripts — one to build HashiCorp Packer images, one to run `terraform plan` and `terraform apply`, and one to automatically update the value of a variable in a Terraform tfvars or Terragrunt HCL file.
 
 If you need to run a custom script in the `deploy-runner`, you must fork the image code, add an additional line to copy your script into directory designated by the `trigger_directory` argument. Then, you will need to rebuild the Docker image, push to ECR, then update your Pipelines deployment following the steps in [Updating Pipelines](./updating.md).
 
```

Comment:
Seems like a good candidate for wrapping in an admonition for visual callout.

Reply:
In this case I think putting this in an admonition might draw the eye away from the first paragraphs that set up context. WDYT?

Reply:
That makes sense. I am okay with leaving it as it is.